R/resolve-gene-aliases.R

```diff
@@ -9,6 +9,7 @@
 #' and replaces those aliases with the accepted (most recent) gene name.
 #' Function uses `gnomeR::impact_alias_table` by default as reference for
```

Comment:
let's add checks for multiple aliases in one string and throw warning if not. We will add docs to user about structure the alias table should take.